app/cloud.go

```diff
@@ -133,14 +133,40 @@ func (a *App) getSysAdminsEmailRecipients() ([]*model.User, *model.AppError) {
 	return a.GetUsersFromProfiles(userOptions)
 }
 
+func getCurrentPlanName(a *App) (string, error) {
+	subscription, err := a.Cloud().GetSubscription("")
+	if err != nil {
+		return "", model.NewAppError("SendDelinquencyEmail", "app.cloud.get_subscription.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
+	if subscription == nil {
+		return "", model.NewAppError("SendDelinquencyEmail", "app.cloud.get_subscription.app_error", nil, "", http.StatusInternalServerError)
+	}
+
+	products, err := a.Cloud().GetCloudProducts("", false)
+	if err != nil {
+		return "", model.NewAppError("SendDelinquencyEmail", "app.cloud.get_cloud_products.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
+	if products == nil {
+		return "", model.NewAppError("SendDelinquencyEmail", "app.cloud.get_cloud_products.app_error", nil, "", http.StatusInternalServerError)
+	}
+
+	planName := getCurrentProduct(subscription.ProductID, products).Name
+	return planName, nil
+}
+
 func (a *App) SendPaymentFailedEmail(failedPayment *model.FailedPayment) *model.AppError {
 	sysAdmins, err := a.getSysAdminsEmailRecipients()
 	if err != nil {
 		return err
 	}
 
+	planName, err2 := getCurrentPlanName(a)
```

Comment:
I am guessing we're doing err2 because of types mismatch. Returning *model.AppError get rid of err2